packages/web-react/src/components/Toast/stories/UncontrolledToast.stories.tsx

```diff
@@ -1,19 +1,22 @@
-import React from 'react';
 import { Markdown } from '@storybook/blocks';
 import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
 
-import { ToastColorType, UncontrolledToastProps } from '../../../types';
+import { ToastProvider, UncontrolledToast } from '..';
 import { AlignmentX, EmotionColors } from '../../../constants';
+import { ToastColorType, UncontrolledToastProps } from '../../../types';
 import { Button } from '../../Button';
 import ReadMe from '../README.md';
+import { TOAST_AUTO_CLOSE_INTERVAL } from '../constants';
 import { useToast } from '../useToast';
-import { ToastProvider, UncontrolledToast } from '..';
 
 interface UncontrolledToastPlaygroundProps extends UncontrolledToastProps {
```

Comment:
🤔 why do we need this?

Reply:
If you mean the type, it's because the `show()` method expects extra props for this storybook demo compared to an uncontrolled component. I made a small change by extending it with `ToastItem`.

Reply:
ok